console/src/Globals.js

```diff
@@ -25,8 +25,8 @@ const globals = {
   dataApiUrl: checkExtraSlashes(window.__env.dataApiUrl),
   devDataApiUrl: window.__env.devDataApiUrl,
   nodeEnv: window.__env.nodeEnv,
-  accessKey: window.__env.accessKey,
-  isAccessKeySet: window.__env.isAccessKeySet,
+  adminSecret: window.__env.adminSecret,
```

Comment:
cli sends this adminSecret env. But console needs to check for accessKey env for older versions of cli.